plugin.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "Database Items",
-    "description": "Provide database access via items.",
+    "name": "Database Assetstore",
```

Comment:
Okay.  If we care, a few other plugins are also title cased (Item Previews and Google Analytics (if it counts, since that could be viewed as a proper name)).